substrate/primitives/keyring/src/sr25519.rs

```diff
@@ -159,14 +159,20 @@ impl FromStr for Keyring {
 
 	fn from_str(s: &str) -> Result<Self, <Self as FromStr>::Err> {
 		match s {
-			"alice" => Ok(Keyring::Alice),
-			"bob" => Ok(Keyring::Bob),
-			"charlie" => Ok(Keyring::Charlie),
-			"dave" => Ok(Keyring::Dave),
-			"eve" => Ok(Keyring::Eve),
-			"ferdie" => Ok(Keyring::Ferdie),
-			"one" => Ok(Keyring::One),
-			"two" => Ok(Keyring::Two),
+			"Alice" => Ok(Keyring::Alice),
```

Comment:
I would keep old values:

```suggestion
			"Alice"|"alice" => Ok(Keyring::Alice),
```

Reply:
> I would keep old values:

The old values is not consistent with enum definition and it is also not idiomatic in English. They look weird.

Reply:
How do you know there are not used? You can break things with this change. Please keep them.